build/e2e.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -ex
+
+DB_NAME=e2e_tests
+PORT=5432
+VERSION=$(./scripts/compute_build_number.sh)
+
+dpkg -i "./packages/$VERSION/algorand-indexer_${VERSION}_amd64.deb"
+
+/etc/init.d/postgresql start
+sudo -u postgres bash -c "psql -c \"CREATE ROLE $USER WITH SUPERUSER CREATEDB LOGIN ENCRYPTED PASSWORD '$USER';\""
+sudo -u postgres bash -c "psql -c \"CREATE DATABASE $DB_NAME;\""
+
+python3 misc/e2etest.py --connection-string "host=localhost port=$PORT dbname=$DB_NAME sslmode=disable user=$USER password=$USER"
```

Comment:
Why not let it create a dbname as usual? Is this something that Docker needs? If this is a test that should be run inside Docker for testing then we shouldn't reference build/e2e.sh from the Makefile test target which will probably be used by developers not inside docker.

Reply:
I passed it the connection string b/c it needed the password for the user.  I was thinking about how a dev would test this, as well, for the same reasons you bring up.

It definitely needs some work to achieve a painless process for both our testing and a local dev.

Reply:
On my dev system with postgres installed $USER is a postgres super user so misc/e2etest.py creating the database just works. I'd expect line 12 above to take care of that.